config/crd/bases/appstudio.redhat.com_integrationtestscenarios.yaml

```diff
@@ -317,8 +319,10 @@ spec:
                   name:
                     type: string
                   type:
-                    description: EnvironmentType currently indicates whether an environment
-                      is POC/Non-POC, see API doc for details.
+                    description: 'DEPRECATED: EnvironmentType should no longer be
```

Comment:
was this supposed to be part of this PR?

Reply:
I tried to not bump the application-api but it does it automatically by updating the release-service. So yes, this change it required to match the application-api bump.

Reply:
I think it should be fine to have the updated field IMO.